app/assets/stylesheets/components/internal_link.scss

```diff
@@ -10,7 +10,7 @@
       content: "\f0c1";
       font-family: "Font Awesome 5 Free";
       font-size: .75em;
-      left: -1.2em;
+      left: -1.8em;
```

Comment:
I've been QA'ing in the review app and it seems like this is a bit too much space between the icon and the text. -1.5 or -1.6 seems like the sweet spot.